github/pulls_reviews.go

```diff
@@ -50,3 +62,49 @@ func (s *PullRequestsService) ListReviews(owner string, repo string, number int)
 
 	return *reviews, resp, err
 }
+
+// GetReview fetches the specified pull request review.
+//
+// GitHub API docs: https://developer.github.com/v3/pulls/reviews/#get-a-single-review
+func (s *PullRequestsService) GetReview(owner string, number int, id int) (*PullRequestReview, *Response, error) {
+	u := fmt.Sprintf("repos/%v/pulls/%d/reviews/%d", owner, number, id)
```

Comment:
Thanks for noticing this @shurcooL! Even I was suspicious at the time of writing that method however, didn't delve deeper.

I have contacted GitHub about the error in their documentation. I will update the endpoint in our code.